sql/core/src/main/scala/org/apache/spark/sql/execution/ShuffledRowRDD.scala

```diff
@@ -132,8 +128,6 @@ class ShuffledRowRDD(
   private[this] val part: Partitioner =
     new CoalescedPartitioner(dependency.partitioner, partitionStartIndices)
 
-  override def getDependencies: Seq[Dependency[_]] = List(dependency)
```

Comment:
Without this it will return nothing, because the superclass is passed Nil for deps.